libpod/container_internal.go

```diff
@@ -1734,6 +1734,15 @@ func (c *Container) chownVolume(volumeName string) error {
 		if err := os.Lchown(mountPoint, uid, gid); err != nil {
 			return err
 		}
+
+		st, err := os.Lstat(filepath.Join(c.state.Mountpoint, v.Dest))
```

Comment:
Could you add a comment here? Maybe also mention #10188 for historical reasons.

Reply:
Sure,